### PR TITLE
Feature Carousel: Removed Resource from Bean and refactored code

### PR DESCRIPTION
**Pull request by pradeeprajput**

### CODE REVIEW

File: core/src/main/java/com/taylorsuniversity/models/FeatureCarouselModel.java [hunk 1]
```diff
@@ -44,11 +46,52 @@
             featureCarouselItems = new ArrayList<>();
 
             for (Resource feature : carouselItems) {
-                logger.debug("Tile items are :", feature);
-                featureCarouselItems.add(feature.adaptTo(FeatureCarouselModelBean.class));
+                logger.debug("Caroursel item is :", feature);
```
Comment (bsinghal1) [inline core/src/main/java/com/taylorsuniversity/models/FeatureCarouselModel.java]: logger.debug("Caroursel item is : {}", feature.getPath());

File: core/src/main/java/com/taylorsuniversity/models/FeatureCarouselModel.java [hunk 2]
```diff
@@ -44,11 +46,52 @@
...
+    private void setFeatureItems(final ValueMap vm) {
+
+        String desktopImage = StringUtils.EMPTY;
+        String mobileImage = StringUtils.EMPTY;
+        String title = StringUtils.EMPTY;
+        String text = StringUtils.EMPTY;
+        String ctaText = StringUtils.EMPTY;
+        String ctaLink = StringUtils.EMPTY;
+
+        if (null != vm.get("desktopImage").toString()) {
```
Comment (bsinghal1) [inline core/src/main/java/com/taylorsuniversity/models/FeatureCarouselModel.java]: check for null without ".toString()"

Comment (pradeeprajput) [reply core/src/main/java/com/taylorsuniversity/models/FeatureCarouselModel.java]: Done

File: core/src/main/java/com/taylorsuniversity/models/FeatureCarouselModel.java (hunk 1 above)
Comment (pradeeprajput) [reply core/src/main/java/com/taylorsuniversity/models/FeatureCarouselModel.java]: done

**Pull request merged**